docs/examples/basic_conceptual_guide/Part_4-Inference_Acceleration/Readme.md

```diff
@@ -0,0 +1,189 @@
+# Accelerating Inference for Deep Learning Models
+
+Model acceleration is a complex nuanced topic. The viablity of techniques like graph optimizations for models, pruning, knowledge distalation, quantization, and more, highly depend on the structure of the model. Each of these topics are vast fields of research in their own right and building custom tools requires massive engineering investment. It is reasonable to assume that for challenges as complex and important as the aforementioned, a plethora of tools and SDKs would be available. A discussion of the merits and challenges of each tool comes with many "in-general"(s) and caviats. Rather than having an exhaustive outline of the ecosystem, for brevity and objectivity, this disucssion will be focused on the tools and features which are recommended to use while deploying models using the Triton Inference Server.
```

Comment:
```suggestion
Model acceleration is a complex nuanced topic. The viability of techniques like graph optimizations for models, pruning, knowledge distillation, quantization, and more, highly depend on the structure of the model. Each of these topics are vast fields of research in their own right and building custom tools requires massive engineering investment. 

Rather than having an exhaustive outline of the ecosystem, for brevity and objectivity, this discussion will be focused on the tools and features which are recommended to use while deploying models using the Triton Inference Server.
```